cmd/lambda/main.go

```diff
@@ -45,14 +50,26 @@ func newStatikStore(statikPath string, logger logrus.FieldLogger) (*store.Store,
 func listenAndServe() error {
 	logger = logrus.New()
 
+	var stores []store.Store
+
 	statikStore, err := newStatikStore("/plugins.json", logger)
 	if err != nil {
 		return err
 	}
+	stores = append(stores, statikStore)
+
+	if upstreamURL != "" {
+		upstreamStore, err := store.NewProxy(upstreamURL, logger)
+		if err != nil {
+			return errors.Wrap(err, "failed to initialize upstream store")
+		}
+
+		stores = append(stores, upstreamStore)
+	}
 
 	router := mux.NewRouter()
 	api.Register(router, &api.Context{
```

Comment:
It seams counter intuitive to to generate a `store.NewMerged` using only one `store`. 

What about declaring a `store.Store` first and then make it an `MergedStore` if needed:
```suggestion
	var newStore store.Store
	newStore, err := newStatikStore("/plugins.json", logger)
	if err != nil {
		return err
	}

	if upstreamURL != "" {
		upstreamStore, err := store.NewProxy(upstreamURL, logger)
		if err != nil {
			return errors.Wrap(err, "failed to initialize upstream store")
		}

		newStore = store.NewMerged(logger, newStore, upstreamStore)
	}

	router := mux.NewRouter()
	api.Register(router, &api.Context{
		Store:  newStore,
		Logger: logger,
	})
```